plugin/trino-jmx/src/main/java/io/trino/plugin/jmx/JmxMetadata.java

```diff
@@ -152,7 +152,7 @@ private JmxTableHandle getJmxTableHandle(SchemaTableName tableName)
                     .sorted(comparing(JmxColumnHandle::getColumnName))
                     .collect(toImmutableList());
 
-            return new JmxTableHandle(tableName, objectNames.stream().map(ObjectName::toString).collect(toImmutableList()), columns, true, TupleDomain.all());
+            return new JmxTableHandle(tableName, objectNames.stream().map(ObjectName::getCanonicalName).collect(toImmutableList()), columns, true, TupleDomain.all());
```

Comment:
I'm not sure if we need to use `getCanonicalName` name here as it represents the name of the object and it is a part of the data.

Reply:
`getCanonicalName` will returns the canonical form of the name; that is, a string representation where the properties are sorted in lexical order. 
`toString ` will return the name and the key properties in the returned string will be in the same order as in the argument to the constructor.
![image](https://user-images.githubusercontent.com/37698246/221138640-ca3d1a7b-567c-4853-8aef-64369c835213.png)

Reply:
I don't think it matters which table name is used, what matters is that we need to be consistent with table name in `JmxHistoricalData`

Reply:
cc @Praveen2112